cg/cli/utils.py

```diff
@@ -12,3 +14,7 @@ def is_case_name_allowed(name: str) -> bool:
     """Returns true if the given name consists only of letters, numbers and dashes."""
     allowed_pattern: re.Pattern = re.compile("^[A-Za-z0-9-]+$")
     return bool(allowed_pattern.fullmatch(name))
+
+
+def click_context_setting_max_content_width() -> dict[str, int]:
+    return {"max_content_width": shutil.get_terminal_size().columns - 10}
```

Comment:
```suggestion
MAX_CONTENT_WIDTH :dict[str, int] = {"max_content_width": shutil.get_terminal_size().columns - 10}
```
I think this could be a constant. Right @islean ?